src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -332,11 +420,11 @@ def test_req_no_inputs__fail(self):
         ]
 
         for msg_type in functions:
-            req_dict = make_comm_msg(msg_type, None, False)
+            req_dict = make_comm_msg(msg_type, None)
             err = JobRequestException(ONE_INPUT_TYPE_ONLY_ERR)
             with pytest.raises(type(err), match=str(err)):
                 self.jc._handle_comm_message(req_dict)
-            self.check_error_message(req_dict, err)
+            check_error_message(self.jc, req_dict, err)
```

Comment:
all these `check_` functions take the JobComm object as the first argument (previously they used `self.jc` to get it) so these function calls are changed slightly